### ISSUE TITLE
[USABILITY]: 526 V2 - TTY should be a link in the PTSD help pages

### ISSUE THREAD

**Issue by peterhill-oddball** [department-of-veterans-affairs/va.gov-team#1087]
Migrated from vets.gov issue [#16353](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16353)

## Issue
The TTY phone number for users who are hard of hearing is only text, and would be more beneficial if it was a link like the other Veterans resources. Screenshot attached below.

## Acceptance Criteria
* As a keyboard user, I would like the TTY phone number to be a link
* As a screen reader user, I would like the TTY phone number to announce itself as a link

---
![screen shot 2019-01-17 at 3 56 48 pm](https://user-images.githubusercontent.com/934879/51352007-fd037680-1a71-11e9-924b-d908ad2a3829.png)

**Comment by peterhill-oddball** [comment]
Prior to migration, this issue was part of an epic [](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15434)
